Rakefile

```diff
@@ -4,6 +4,7 @@ require 'English'
 XCODE_WORKSPACE = 'Simplenote.xcworkspace'
 XCODE_SCHEME = 'Simplenote'
 XCODE_CONFIGURATION = 'Debug'
+LOCAL_PATH = "vendor/bundle"
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/simplenote-ios/blob/97ee2b721bba1d6abdd082b67db8ccc7e826004d/Rakefile#L7"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>